src/views/RecIM/components/Listing/index.js

```diff
@@ -33,14 +35,37 @@ const TeamListing = ({ activityID, teamID }) => {
 
 // We could also use ParticipantID (not student ID) if we have that and prefer it to AD_Username
 const ParticipantListing = ({ username }) => {
+  const [avatar, setAvatar] = useState('');
+
+  // const [name, setName] = useState({
+  //   firstname: '',
+  //   lastname: '',
+  // });
```

Comment:
I think the user hook has a fullName field that we could use for a clean first/last name. I did a little bit of digging and we can't really get the gordon id very easily from the hook so we might need to add an AD_Username to a Participant viewmodel.

Reply:
Good note for the future!